docs/package.json

```diff
@@ -55,6 +55,7 @@
     "ramda": "0.27.1",
     "raw-loader": "4.0.2",
     "remark-admonitions": "1.2.1",
+    "remarkable": "2.0.1",
```

Comment:
Please add the markdown generation task to the `npm run gen` task and make sure to test if it is generating correctly when being applied.